<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservationDialog</class>
 <widget class="QDialog" name="ReservationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RÃ©servation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboClient"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboVol"/>
   </item>
   <item>
    <widget class="QDateEdit" name="dateReservation">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboStatut"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Enregistrer</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
